### DISCUSSION TITLE
Time Spent Alert

### DISCUSSION THREAD

**Discussion by keshavashiya** [dailydotdev/daily#518] [Feature Requests]
Hi Team,

Can we have **`Time Spent Alert`** so we've been notified if we're spending more time on it, since while reading blogs we forgot to get back to work and this leads to miss deadlines.

Thanks

**Comment by idoshamun** [comment]
Hi @keshavashiya,

Thanks for the suggestion. Unfortunately, at the moment it's out of scope.
